### DISCUSSION TITLE
[Question][Hw5] About cmp in sign domain

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#601] [Q&A]
I saw #128, and if I ask once more to clarify my understanding, does cmp in sign domain returns Pos if true, and Zero if false? For example, does cmp < Zero Pos returns Pos because Zero<Pos is true? And also I can't figure out what to do if I can't know whether the condition is true or not, like cond < Pos Pos. Do I return Zero in this situation? Thank you:)

**Comment by RiceBiscuits** [comment]
It is up to you. 

please refer this issue page.
https://github.com/prosyslab-classroom/cs448-2021-spring/issues/134#issuecomment-848579697
